dbx/srv-polling.rst

```diff
@@ -0,0 +1,11 @@
+To use DNS service discovery to look up the DNS SRV record of the service you're connecting to,
+specify the SRV connection format in your connection string. Additionally, if you enable
+the SRV connection format, the {+driver-short+} automatically re-scans for new hosts without
+having to change the client configuration.
```

Comment:
this syntax is a little odd. it makes it sound like the driver (subject of the sentence) is the thing that has to or doesn't have to change the config, even though it's the user who does that. what if we just ditch the last part?

```suggestion
the SRV connection format, the {+driver-short+} automatically re-scans for new hosts.
```

Reply:
I added an extra sentence that's hopefully more clear, let me know what you think